projects/admin/src/app/record/search-view/patron-transaction-event-search-view/payments-data/pie/payment-data-pie.component.html

```diff
@@ -0,0 +1,37 @@
+<!--
+  RERO ILS UI
+  Copyright (C) 2019-2022 RERO
+  Copyright (C) 2019-2022 UCLouvain
+
+  This program is free software: you can redistribute it and/or modify
+  it under the terms of the GNU Affero General Public License as published by
+  the Free Software Foundation, version 3 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+  GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+<div class="chart-container">
+  <div class="payment-pie-chart">
+    <ngx-charts-pie-chart
+      [view]="view"
+      [results]="values"
+      [doughnut]="true"
+      [arcWidth]="0.7"
+      [animations]="animation"
+      [legend]="legendDisplay"
+      [legendPosition]="legendPosition"
+      [legendTitle]="legendTitle"
+    >
+      <ng-template #tooltipTemplate let-model="model">
+        <span class="tooltip-label">{{ model.label }}</span>
```

Comment:
translate ?

Reply:
this has be done into the component.ts because it doesn't work as expected if I do it into the template :(